message_parser/src/main/java/cumulus_sled/message_parser/IMessageAdapter.java

```diff
@@ -0,0 +1,15 @@
+package cumulus_sled.message_parser;
```

Comment:
This should be changed to cumulus_message_adapter.

Reply:
done and all the other renaming is done.